networkx/algorithms/tree/tests/test_branchings.py

```diff
@@ -28,6 +28,28 @@
     [0, 0, 0, 0, 0, 0, 0, 18, 0],  # 8
 ], dtype=int)
 
+# Two copied of the graph from the original paper as disconnected components
+G_big_array = np.array([
+    # 0   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17
+    [ 0,  0, 12,  0, 12,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0],  # 0
+    [ 4,  0,  0,  0,  0, 13,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0],  # 1
+    [ 0, 17,  0, 21,  0, 12,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0],  # 2
+    [ 5,  0,  0,  0, 17,  0, 18,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0],  # 3
+    [ 0,  0,  0,  0,  0,  0,  0, 12,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0],  # 4
+    [ 0,  0,  0,  0,  0,  0, 14,  0, 12,  0,  0,  0,  0,  0,  0,  0,  0,  0],  # 5
+    [ 0,  0, 21,  0,  0,  0,  0,  0, 15,  0,  0,  0,  0,  0,  0,  0,  0,  0],  # 6
+    [ 0,  0,  0, 19,  0,  0, 15,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0],  # 7
+    [ 0,  0,  0,  0,  0,  0,  0, 18,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0],  # 8
+    [ 0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0, 12,  0, 12,  0,  0,  0,  0],  # 9
+    [ 0,  0,  0,  0,  0,  0,  0,  0,  0,  4,  0,  0,  0,  0, 13,  0,  0,  0],  # 10
+    [ 0,  0,  0,  0,  0,  0,  0,  0,  0,  0, 17,  0, 21,  0, 12,  0,  0,  0],  # 11
+    [ 0,  0,  0,  0,  0,  0,  0,  0,  0,  5,  0,  0,  0, 17,  0, 18,  0,  0],  # 12
+    [ 0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0, 12,  0],  # 13
+    [ 0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0, 14,  0, 12],  # 14
+    [ 0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0, 21,  0,  0,  0,  0,  0, 15],  # 15
+    [ 0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0, 19,  0,  0, 15,  0,  0],  # 16
+    [ 0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0, 18,  0],  # 17
+], dtype=int)
```

Comment:
A minor suggestion to compose this from `G_array` - the main motivation for doing so would be to reduce the chance that something went wrong from having to explicitly type out the array twice. Not really a big deal though!

```suggestion
# Two copies of the graph from the original paper as disconnected components
G_big_array = np.zeros(np.array(G_array.shape) * 2, dtype=int)
G_big_array[:G_array.shape[0], :G_array.shape[1]] = G_array
G_big_array[G_array.shape[0]:, G_array.shape[1]:] = G_array
```